bears/jinja2/Jinja2Bear.py

```diff
@@ -0,0 +1,337 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.Diff import Diff
+
+
+def generate_spacing_diff(file, filename, line, line_number,
+                          match_object, required_spacing):
+    '''
+    Generate a diff for incorrectly spaced control or variable tags.
```

Comment:
I'm sorry, I fail to see how this could be done in a readable way. Also, doctest aren't being run by the testrunner, so what's the point?

Or are you talking about like a psudo-doctest example?